src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.cs

```diff
@@ -124,7 +124,7 @@ internal bool HasReferenceOrLoadDirectives
                 if (Options.Kind == SourceCodeKind.Script)
                 {
                     var compilationUnitRoot = GetCompilationUnitRoot();
-                    return compilationUnitRoot.GetReferenceDirectives().Count > 0 || compilationUnitRoot.GetLoadDirectives().Count > 0;
+                    return compilationUnitRoot.HasReferenceOrLoadDirectives();
```

Comment:
There's also a `GetCompilationUnitRoot().GetReferenceDirectives().Count > 0` in the property above this, though it only triggers for scripts.

Reply:
updated that as well.